### ISSUE TITLE
Annotation search tab would be more useful if search includes Comments, DBXrefs, Etc.

### ISSUE THREAD

**Issue by erasche** [GMOD/Apollo#1070]
![utvalg_861](https://cloud.githubusercontent.com/assets/458683/15614742/f14356d0-2429-11e6-9aee-dc8349976ded.png)

**Comment by nathandunn** [comment]
Makes sense.  Would like to get input from @monicacecilia  et al on this as well.  

Would be a pain to do that type of denormalization within a DB . . maybe something like an embedded (or otherwise) elastic-search.
